charts/icm-as/templates/_probes.tpl

```diff
@@ -12,9 +12,12 @@ startupProbe:
   initialDelaySeconds: {{ .Values.probes.startup.initialDelaySeconds | default 60 }}
   periodSeconds: {{ .Values.probes.startup.periodSeconds | default 10 }}
 livenessProbe:
-  httpGet:
-    path: /status/LivenessProbe
-    port: mgnt
+  exec:
+    command:
+      - /bin/bash
+      - -c
+      - |
+        [ -f "/tmp/liveness-status" ] || curl -f http://localhost:7744/status/LivenessProbe >/dev/null 2>&1
```

Comment:
Could management connector port variable `mgnt` interpolation be used here instead of referencing port `7744` directly?

Reply:
You may have a look..